webclient/src/views/search/view-search.js

```diff
@@ -1,18 +1,22 @@
 function searchNavigateTo(to, from, next, component) {
     navigatum.beforeNavigate(to, from);
 
-    cached_fetch.fetch(navigatum.api_base + 'search/' + window.encodeURIComponent(to.params.query) +
-                       '?limit_buildings=10&limit_rooms=30&limit_all=30',
-                                   {cache: "no-cache"})
+    const params = new URLSearchParams()
+    params.append('q', to.query.q)
+    params.append('limit_buildings','10')
+    params.append('limit_rooms','30')
+    params.append('limit_all','30')
+
+    cached_fetch.fetch(navigatum.api_base + 'search?' + params.toString(), {cache: "no-cache"})
```

Comment:
I am not, since IE/Baidu Browser are not an issue,
Opera Mini may be, but my assumption is that with current days data plans their adoption rate in europe is not that high..

https://caniuse.com/urlsearchparams